runelite-client/src/main/java/net/runelite/client/callback/Hooks.java

```diff
@@ -72,6 +74,12 @@
 
 	public static void clientMainLoop(Client client, boolean arg1)
 	{
+		client.clearItemBorderColors();
```

Comment:
What is the point of this event? You can just create the subscription to gameTick or scheduled method in respective plugin